src/components/molecules/applicant-list-pay/applicant-list-pay.tsx

```diff
@@ -31,7 +31,13 @@ export const ApplicantListPay = (props: ApplicantListPayProps): JSX.Element => {
   const applicantJSX = (applicant: Applicant) => {
     return (
       <div key={applicant.id} className={css.applicantContainer}>
-        <ProfileView name={applicant.name} location={applicant.category} img={applicant.image} type="users" />
+        <ProfileView
+          name={applicant.name}
+          username={applicant?.username}
+          location={applicant.category}
+          img={applicant.image}
+          type="users"
```

Comment:
Question: Shouldn't this value come from API?

Reply:
i didn't change anything here i just add the username props.

Reply:
BTW i think you set it fixed as users cuz all applicants are users :)